bokeh/util/tests/test_dependencies.py

```diff
@@ -14,3 +14,11 @@ def test_required_fail():
     with pytest.raises(RuntimeError) as excinfo:
         dep.import_required('bleepbloop', 'nope')
     assert 'nope' in str(excinfo.value)
+
+def test_detect_phantomjs_success():
+    assert dep.detect_phantomjs() is not None
+
+def test_detect_phantomjs_fail(monkeypatch):
```

Comment:
The ``RuntimeError`` exception isn't being raised. For some reason the monkeypatching isn't working.